shell/apps/remote-planning/index.js

```diff
@@ -7,6 +7,7 @@ import {ArcFactory} from './arc-factory.js';
 const userid = '-LMtek9LSN6eSMg97nXV'; // Cletus
 
 const manifest = `
+  import 'https://$artifacts/Arcs/Arcs.recipes'
```

Comment:
I don't think you should do this, Arcs.recipes is not part of `canonical.manifest` on purpose.

Reply:
Oh, sorry, this is a remainder or my failed attempt to make launcher recipe work. Removed.